src/test/typescript-service-helpers.ts

```diff
@@ -2169,44 +2169,22 @@ export function describeTypeScriptService(createService: TypeScriptServiceFactor
 			//  * the end of the snippet. Placeholders with equal identifiers are linked,
 			//  * that is typing in one will update others too.
 			assert.equal(result.isIncomplete, false);
-			assert.sameDeepMembers(result.items, [
-				{
-					label: 'bar',
-					kind: CompletionItemKind.Method,
-					documentation: 'bar doc',
-					sortText: '0',
-					insertTextFormat: InsertTextFormat.Snippet,
-					insertText: 'bar(${1:num})',
-					detail: '(method) A.bar(num: number): number'
-				},
-				{
-					label: 'baz',
-					kind: CompletionItemKind.Method,
-					documentation: 'baz doc',
-					sortText: '0',
-					insertTextFormat: InsertTextFormat.Snippet,
-					insertText: 'baz(${1:num})',
-					detail: '(method) A.baz(num: number): string'
-				},
-				{
-					label: 'foo',
-					kind: CompletionItemKind.Method,
-					documentation: 'foo doc',
-					sortText: '0',
-					insertTextFormat: InsertTextFormat.Snippet,
-					insertText: 'foo()',
-					detail: '(method) A.foo(): void'
-				},
-				{
-					label: 'qux',
-					kind: CompletionItemKind.Property,
-					documentation: 'qux doc',
-					sortText: '0',
-					insertTextFormat: InsertTextFormat.Snippet,
-					insertText: 'qux',
-					detail: '(property) A.qux: number'
-				}
-			]);
```

Comment:
why is this assertion removed?

Reply:
The snippets logic is only used when resolving completion items, and I thought it would be a lot of logic to resolve all the items from the initial response.
Now that I think of it, it's probably not that bad - I'll give it a try!

Reply:
I would leave this test as is (just change the objects) and add a separate test for the resolving